knowledge-base/pmd/pmd-github-action/action-security.yml

```diff
@@ -0,0 +1,8 @@
+name: 'PMD'
+github-token:
+  action-input:
+    input: token
+    is-default: false
+  permissions:
```

Comment:
This needs PR read: https://github.com/pmd/pmd-github-action/blob/79475d6cfdb0eced16531b676ee81d333620d323/lib/util.js#L117
Contents read reason is this: https://github.com/pmd/pmd-github-action/blob/79475d6cfdb0eced16531b676ee81d333620d323/lib/util.js#L143